saltyrtc/server/events.py

```diff
@@ -1,27 +1,67 @@
 import collections
 import enum
-from typing import List
+from typing import (
+    Callable,
+    Coroutine,
+    Dict,
+    List,
+    Optional,
+)
 
-try:
-    from collections.abc import Coroutine
-except ImportError:  # python 3.4
-    from backports_abc import Coroutine
+__all__ = (
+    'DisconnectedData',
+    'EventData',
+    'EventCallback',
+    'Event',
+    'EventRegistry',
+)
 
-__all__ = ('Event',)
+
+# Types
+DisconnectedData = int
+EventData = Optional[DisconnectedData]
```

Comment:
I'd use a union here, since more event data types might be added in the future. The `Optional` results in the same, but semantically I think the union would be better.

(But I have no strong opinon on that as long as typechecks are correct.)